sw/device/tests/BUILD

```diff
@@ -1108,6 +1110,65 @@ opentitan_test(
     ],
 )
 
+# hash table for lc_state name and value.
+# 'name' is picked for pre generated otp image
+# 'value' is copied from
+#   https://cs.opensource.google/opentitan/opentitan/
+#   +/master:rules/const.bzl;drc=30f44025dfb4bc2bd328a99a0a810c3189fb2544;l=29
+_FLASH_CTRL_INFO_ACCESS_LC_STATES = {
+    "test_unlocked0": 0x02108421,
+    "dev": 0x21084210,
+    "prod": 0x2318c631,
+    "prod_end": 0x25294a52
+}
+
+load(
+    "//rules:opentitan.bzl",
+    "RSA_ONLY_KEY_STRUCTS",
+)
+load(
+    "//rules/opentitan:defs.bzl",
+    "rsa_key_for_lc_state",
+)
+
+# This loop will create following sival bazel target
+#  flash_ctrl_info_access_lc_test_unlocked0
+#  flash_ctrl_info_access_lc_test_dev
+#  flash_ctrl_info_access_lc_test_prod
+#  flash_ctrl_info_access_lc_test_prod_end
+
```

Comment:
You can remove this. Use the following to create a single test target for the generated tests (add this after the `opentitan_test` rule:

```python
test_suite(
    name = "flash_ctrl_info_access_lc_states",
    tests = ["flash_ctrl_info_access_lc_{}".format(lc_state) for lc_state, _ in _FLASH_CTRL_INFO_ACCESS_LC_STATES],
)
```

Then you will be able to run the test as follows: 

```console
bazel test  \
 --define DISABLE_VERILATOR_BUILD=true \
 --test_tag_filters=cw310  \
 --test_output=streamed   \
 --define bitstream=gcp_splice   \
 --cache_test_results=no \
 //sw/device/tests:flash_ctrl_info_access_lc_states
```

If you decide to leave the comment in, then update the naming for dev, prod and prod_end to remove `_test_` after `_lc`.

Reply:
added.